cmd/identity/main.go

```diff
@@ -128,6 +137,11 @@ func cmdAuthorize(cmd *cobra.Command, args []string) error {
 	ctx := process.Ctx(cmd)
 
 	serviceDir := serviceDirectory(args[0])
+
+	if writable, err := utils.IsWritable(serviceDir); !writable && err != nil {
```

Comment:
if err != nil but it is writable, I think that err would still need to be returned

Reply:
Should've been `||` thanks